crates/next-core/js/src/dev/hmr-client.ts

```diff
@@ -245,17 +270,21 @@ const CATEGORY_ORDER = [
   "other",
 ];
 
-function handleSocketMessage(msg: ServerMessage) {
-  msg.issues.sort((a, b) => {
+function sortIssues(issues: Issue[]): Issue[] {
+  return issues.sort((a, b) => {
```

Comment:
Let's not return anything to make it clear we're mutating `issues`.

Reply:
Probably better yes, just mirrored the `Array.sort` API